src/client/core/prevent-real-events.js

```diff
@@ -0,0 +1,74 @@
+import { utils, eventSandbox } from './deps/hammerhead';
+
+import { get, hasDimensions } from './utils/style';
+import { filter } from './utils/array';
+import { isShadowUIElement, isWindow, getParents } from './utils/dom';
+
+var browserUtils   = utils.browser;
+var listeners      = eventSandbox.listeners;
+var eventSimulator = eventSandbox.eventSimulator;
+
+const PREVENTED_EVENTS = [
+    'click', 'mousedown', 'mouseup', 'dblclick', 'contextmenu', 'mousemove', 'mouseover', 'mouseout',
+    'touchstart', 'touchmove', 'touchend', 'keydown', 'keypress', 'input', 'keyup', 'change', 'focus', 'blur',
+    'MSPointerDown', 'MSPointerMove', 'MSPointerOver', 'MSPointerOut', 'MSPointerUp', 'pointerdown',
+    'pointermove', 'pointerover', 'pointerout', 'pointerup'
+];
+
+const F12_KEY_CODE = 123;
+
+
+function checkBrowserHotkey (e) {
+    // NOTE: Opening browser tools with F12, CTRL+SHIFT+<SYMBOL KEY>
+    // on PC or with OPTION(ALT)+CMD+<SYMBOL KEY> on Mac.
+    return e.shiftKey && e.ctrlKey || e.altKey && e.isMacPlatform || e.keyCode === F12_KEY_CODE;
+}
+
+// NOTE: when test is running, we should block real events (from mouse
+// or keyboard), because they may lead to unexpected test result.

```

Comment:
sorry, missed it!
thanks to @AlexanderMoskovkin for noticing this
```
// NOTE: when tests are running, we should block real events (from mouse
// or keyboard), because they may lead to unexpected test result.
```